libs/designsystem/src/lib/components/toast/services/toast.helper.ts

```diff
@@ -30,9 +29,19 @@ export class ToastHelper {
 
   private getCssClass(messageType: MessageType): string {
     let cssClass = ToastHelper.CSS_CLASS;
+
+    if (messageType === 'danger') {
+      console.warn(
+        `[DEPRECATED] 'danger' message type is deprecated. Use Alerts for critial warnings. Message type will be set as 'warning'`
+      );
+
+      return (cssClass += ' ' + 'warning');
```

Comment:
Consider the complexity of assigning + returning here while only assigning in other case and finally returning.
Maybe just switch the `messageType` to  'warning' and keep existing null-check + return statement...